network/src/main/java/com/android254/droidconKE2020/network/di/Url.kt

```diff
@@ -0,0 +1,9 @@
+package com.android254.droidconKE2020.network.di
+
+/**
+ * 18/03/20
+ * @author bernard
+ */
+object Url {
+    const val BASE_URL = "https://example.com"
```

Comment:
Hey @ben-mathu what do you think about having one object called Constants where we can put all the constants?

Reply:
Yeah, but where will it sit? Should it be in the core module.

Reply:
If the constants are only being used by one feature, i'd place it inside that feature

Reply:
Okay got it.